media_utils/video_puppeteer.py

```diff
@@ -56,12 +39,13 @@ def __init__(self, marionette, url, video_selector='video'):
                                                           'Using first.')
             if len(videos_found) > 0:
                 self.video = videos_found[0]
-                self.marionette.execute_script("log('video element obtained');")
-
-    def get_debug_lines(self):
```

Comment:
Please don't remove `get_debug_lines`; it's used in `media_test_harness/testcase.py`.